pkg/controllers/dynakube/extension/eec/reconciler_test.go

```diff
@@ -672,7 +666,26 @@ func TestUpdateStrategy(t *testing.T) {
 	})
 }
 
-func TestActiveGateTrustedCAInsideEEC(t *testing.T) {
+func TestVolumes(t *testing.T) {
+	t.Run("Custom EEC tls certificate is mounted to EEC", func(t *testing.T) {
+		dk := getTestDynakube()
+		dk.Spec.Templates.ExtensionExecutionController.TlsRefName = "custom-tls"
+		statefulSet := getStatefulset(t, dk)
+
+		expectedVolume := corev1.Volume{
+			Name: httpsCertVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: "custom-tls",
+				},
+			},
+		}
+
+		require.Contains(t, statefulSet.Spec.Template.Spec.Volumes, expectedVolume)
+	})
+}
```

Comment:
could you please move Volume tests from 304-464 to new TestVolumes function ?

Reply:
fixed in 6280db93